src/daemon/org.projectatomic.rpmostree1.xml

```diff
@@ -65,13 +69,21 @@
     </method>
 
     <!-- Array of all deployments in boot order -->
-    <property name="Deployments" type="aa{sv}" access="read"/>
+    <property name="Deployments" type="aa{sv}" access="read">
+      <annotation name="org.qtproject.QtDBus.QtTypeName" value="QList&lt;QVariantMap>"/>
```

Comment:
Is the `&lt;` expected here? Should `>` also be changed to `&gt;`?

Reply:
I've tried using `<` and it breaks somewhere in the code generation. Using `>` is currently working. I've not tried using `&gt;`.